### DISCUSSION TITLE
ath9k when switching from 2.4G to 5G only lower MCS can be used

### DISCUSSION THREAD

**Discussion by piaooo** [svpcom/wfb-ng#355] [General]
When using ath9k ar9580 on channel 14 with MCS 15, and receiving an RSSI of -50 dBm, everything works fine, speed near 70Mbps. However, when I set the channel to 108 using `iw dev wlan0 set channel 108`, all the data received by tcpdump shows as FCS failures.
I want to set a higher MCS to achieve a higher data rate, is this a characteristic of 5G or a limitation of the WiFi card driver?

**Comment by svpcom** [comment]
I don't have ath9k that 5G capable. For 8812au/eu MCS index up to 7 works fine. MCS15 use two spatial streams. Try to use MCS index with one spatial stream. See https://mcsindex.com/ for reference
